src/data/map.js

```diff
@@ -1256,7 +1257,7 @@ const getSearchData = async (lat, lon, id, value, iconStyle) => {
             FROM ingress_portals
             WHERE LOWER(name) LIKE '%${sanitizedValue}%'
             `;
-            useManualDb = true;
+            useManualDb = dbOptions.includes('portal') ? false : true;
```

Comment:
Simplify check (or remove `!` if we import manualdb useFor)
```suggestion
            useManualDb = !dbOptions.includes('portal');
```